src/ResourceManager/StorageSync/ChangeLog.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
@afedyashov the following comments apply to the `AzureRM.StorageSync.psd1` file:

Using the [`AzureRM.Billing.psd1`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Billing/Commands.Billing/AzureRM.Billing.psd1) file as a baseline, here are some of the things that need to change in `AzureRM.StorageSync.psd1`:

- The `RootModule` field needs to be commented-out and empty
- The `Description` file dneeds to be uncommented and filled out
- The `PowerShellVersion` field needs to be uncommented and filled out (with the value of 5.0)
- The `DotNetFrameworkVersion` field needs to be uncommented and filled out (with the value of 4.5.2)
- The `CLRVersion` field needs to be uncommented and filled out (with the value of 4.0)
- The `RequiredModules` filed needs to be uncommented and filled out (with a dependency on `AzureRM.Profile`, as seen in Billing)
- The `RequiredAssemblies` field should only contain extra assemblies that you reference (in this case, it appears that this is none)
- The `NestedModules` field should contain the assembly with the cmdlet code (in this case, `Microsoft.Azure.Commands.Storage.Sync.dll`)
- The `VariablesToExport` field should be commented-out
- The `ReleaseNotes` field should be uncommented and empty

Reply:
This file is auto-generated. I'll perform updates as you require, but these requirements should be as well documented in the guide https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-developer-guide.md

Reply:
RequiredModules - we do not have dependencies. Uncommented, but leaving it empty.